ament_index_cpp/CMakeLists.txt

```diff
@@ -29,6 +29,11 @@ if(BUILD_TESTING)
   if(TARGET ${PROJECT_NAME}_utest)
     target_include_directories(${PROJECT_NAME}_utest PUBLIC include)
     target_link_libraries(${PROJECT_NAME}_utest ${PROJECT_NAME})
+    if(NOT WIN32)
+        set(THREADS_PREFER_PTHREAD_FLAG ON)
+        find_package(Threads REQUIRED)
+        target_link_libraries(${PROJECT_NAME}_utest Threads::Threads)
```

Comment:
The [previous comment](https://github.com/ament/ament_index/pull/19#discussion_r91747408) about the indentation of this block (to use 2 spaces as the rest of the file instead of 4) still applies.